test/framework/integration_framework/fake_peer/behaviour/behaviour.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef INTEGRATION_FRAMEWORK_FAKE_PEER_BEHAVIOUR_HPP_
+#define INTEGRATION_FRAMEWORK_FAKE_PEER_BEHAVIOUR_HPP_
+
+#include <memory>
+#include <vector>
+
+#include "framework/integration_framework/fake_peer/fake_peer.hpp"
+#include "logger/logger.hpp"
+
+namespace integration_framework {
+  namespace fake_peer {
+
+    class Behaviour {
+     public:
+      virtual ~Behaviour();
+
+      /// Enable the behaviour for the given peer
+      void adopt(const std::shared_ptr<FakePeer> &fake_peer);
+
+      /// Disable the behaviour
+      void absolve();
```

Comment:
You definitely don't want to use "[absolve](https://www.multitran.ru/c/m.exe?CL=1&s=absolve&l1=1)" here :)

Please find more suitable method name

Reply:
But I did want to use that term. If you think it is inappropriate, please provide an alternative.

Reply:
![image](https://user-images.githubusercontent.com/36847043/48710171-951ee400-ec18-11e8-8bb9-2113e4fcca32.png)

Reply:
switch off, shutdown, whatever except absolve)